### PR TITLE
Convert beacon tests to pytest (diskusage, load, network_info & ps)

### PR DESCRIPTION
**Pull request by myii**
### What does this PR do?

Convert beacon tests to pytest (diskusage, load, network_info & ps).

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltstack.com/en/master/topics/development/changelog.html
- [x] Tests written/updated

### Commits signed with GPG?
Yes

### CODE REVIEW

File: tests/pytests/unit/beacons/test_load.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+def test_empty_config():
+    config = [{}]
+
+    ret = load.validate(config)
+    assert ret == (False, "Averages configuration is required for load beacon.")
+
+
+@pytest.mark.skipif(
+    salt.utils.platform.is_windows(), reason="os.getloadavg not available on Windows"
+)
```
Comment (s0undt3ch) [inline tests/pytests/unit/beacons/test_load.py:34]:
```suggestion
@pytest.mark.skip_on_windows(reason="os.getloadavg not available on Windows")
```

File: tests/pytests/unit/beacons/test_network_info.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+@pytest.fixture
+def stub_net_io_counters():
+    return {
+        "eth0": namedtuple(
+            "snetio",
+            "bytes_sent bytes_recv \
+                                               packets_sent packets_recv \
+                                               errin errout \
+                                               dropin \
+                                               dropout",
```
Comment (s0undt3ch) [inline tests/pytests/unit/beacons/test_network_info.py:28]:
```suggestion
            "bytes_sent bytes_recv "
            "packets_sent packets_recv "
            "errin errout dropin dropout",
```

File: tests/pytests/unit/beacons/test_ps.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    def __init__(self, _name, pid):
+        self._name = _name
+        self.pid = pid
+
+    def name(self):
+        return self._name
+
+
+@pytest.fixture
+def patch_opts():
```
Comment (s0undt3ch) [inline tests/pytests/unit/beacons/test_ps.py:22]: Personally, I wouldn't use a fixture for this and would just use pass these kwargs to `patch()`, but it's not a merge blocker.

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

**Comment by s0undt3ch** [comment]
Thank You for migrating these tests over to PyTest! :heart:

File: tests/pytests/unit/beacons/test_load.py (hunk 1 above)
Comment (myii) [reply tests/pytests/unit/beacons/test_load.py]: Fixed.

File: tests/pytests/unit/beacons/test_network_info.py (hunk 2 above)
Comment (myii) [reply tests/pytests/unit/beacons/test_network_info.py]: Fixed.

File: tests/pytests/unit/beacons/test_ps.py (hunk 3 above)
Comment (myii) [reply tests/pytests/unit/beacons/test_ps.py]: Fixed -- sorry, I got a bit too caught up in line-by-line translation!

**Review by s0undt3ch — APPROVED** [review]

**Review by s0undt3ch — APPROVED** [review]

**Comment by s0undt3ch** [comment]
Windows failures are known flaky tests

**Pull request merged**